src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -60,7 +60,8 @@ public JsonAdaptedPerson(Person source) {
         skills.addAll(source.getSkills().stream()
                 .map(JsonAdaptedSkill::new)
                 .collect(Collectors.toList()));
-        match = source.getMatch();
+        // During the reconstruction toModelType, have to perform a String.isEmpty() check here
+        match = source.getMatch().orElse("");
```

Comment:
I think for this case, it makes sense to store non-existence value as null. 

During reconstruction `toModelType()`, we check for null instead. 

This is pretty safe too since we can enforce non null value in our Person constructor right?

Reply:
Okaay